recipes/_java.rb

```diff
@@ -2,6 +2,8 @@
 node.default['java']['jdk_version'] = '8'
 node.default['java']['install_flavor'] = 'oracle'
 node.default['java']['set_default'] = true
+node.default['java']['jdk']['8']['x86_64']['url'] = 'https://s3.amazonaws.com/conversocial-chef-public/jdk-8u101-linux-x64.tar.gz'
```

Comment:
I know it will break the opensource version if we don't do this here. but I'd prefer if we didn't publish the location of our mirror for internal use.